### DISCUSSION TITLE
青龙面板更新订阅配置文件被覆盖

### DISCUSSION THREAD

**Discussion by QYG2297248353** [0-8-4/miui-auto-tasks#137] [展示和讲述 ｜Show and tell]
[青龙面板](https://github.com/0-8-4/miui-auto-tasks/wiki#%E9%9D%92%E9%BE%99%E9%9D%A2%E6%9D%BF)
关于更新订阅 配置文件被覆盖：
解决方案：
> 执行到第三步后：
在青龙面板的订阅管理中点击 新建订阅 ，将下方命令参数复制到新建订阅窗口的 名称 中后手动设置定时规则，指定类型为 interval 每 1 天 后点击确定，然后运行订阅并禁用订阅的自动运行，如不禁用将导致配置文件被覆盖。
`ql repo https://github.com/0-8-4/miui-auto-tasks.git miuitask utils|init utils|config master py|yaml`

订阅更新完成后：
> 回到订阅管理 >> 再次编辑刚才新建的订阅 >> 找到 `py|yaml` >> 修改为 `py` >> 保存即可

请先确保第三步的订阅已完成，或者直接使用以下链接拉取
`ql repo https://github.com/0-8-4/miui-auto-tasks.git miuitask utils|init utils|config master py`
拉去完成后进入 脚本管理 >> 找到 `0-8-4_miui-auto-tasks_master` 展开 >> 右上角创建 >> `类型`：空文件 `文件名`：config.yaml `父目录`：0-8-4_miui-auto-tasks_master >> 粘贴本仓库的 `config.yaml` 内容即可

**Comment by lelemka0** [comment]
添加订阅时将依赖文件设置为`utils`，并设置执行后:
`MIUICONFIG="/ql/data/scripts/0-8-4_miui-auto-tasks/config.yaml"; rm $MIUICONFIG; ln -s /ql/data/config/miui-config.yaml $MIUICONFIG;`

此后配置文件即可保留并允许在`青龙webui - 配置文件`中修改。

**Comment by Patxi-X** [comment]
因为配置文件可能需要跟随脚本进行更新，所以建议是禁用自动更新订阅。后续版本重构后会分离账户配置文件，届时会更新wiki
